tests/wsgi_test.py

```diff
@@ -114,8 +115,7 @@ def __call__(self, env, start_response):
 
 
 class IterableApp(object):
-
-    def __init__(self, send_start_response=False, return_val=wsgi.ALREADY_HANDLED):
+    def __init__(self, send_start_response=False, return_val=[]):
```

Comment:
Let's not use a mutable default (just in case `return_val` is modified by something further down the line in the future)

Reply:
Thanks!